skiptests.txt

```diff
@@ -0,0 +1,137 @@
+## No SYNC support
```

Comment:
wondering what this whole file is used for?  i see skip.txt being used, but not this

Reply:
Left over, will remove.